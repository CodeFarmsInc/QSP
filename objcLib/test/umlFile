<?xml version = '1.0' encoding = 'UTF-8' ?>
<XMI xmi.version = '1.2' xmlns:UML = 'org.omg.xmi.namespace.UML'
         timestamp = '1095492568'
  <XMI.header>
    <XMI.documentation>
      <XMI.exporter>CentralControl</XMI.exporter>
      <XMI.exporterVersion>1.0</XMI.exporterVersion>
    </XMI.documentation>
  </XMI.header>
  <XMI.content>
  </XMI.content>
</XMI>
