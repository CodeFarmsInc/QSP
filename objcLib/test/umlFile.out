<?xml version = '1.0' encoding = 'UTF-8' ?>
<XMI xmi.version = '1.2' xmlns:UML = 'org.omg.xmi.namespace.UML'
         timestamp = '1096220808'
  <XMI.header>
    <XMI.documentation>
      <XMI.exporter>CentralControl</XMI.exporter>
      <XMI.exporterVersion>1.0</XMI.exporterVersion>
    </XMI.documentation>
  </XMI.header>
  <XMI.content>
    <UML:Model xmi.id = 'jLib:projectID:1000'
      name = 'jLib:projectID' isSpecification = 'false'
      isRoot = 'false' isLeaf = 'false' isAbstract = 'false'>
      <UML:Namespace.jLib_UML>      </UML:Namespace.jLib_UML>
    </UML:Model>
  </XMI.content>
</XMI>
